.github/templates/main.yml

```diff
@@ -124,64 +153,73 @@ jobs:
       - build-wrappers-android
       - build-wrappers-linux
       - build-wrappers-windows-uwp
+    outputs:
+      nuget_pkg_version: ${{ steps.find-nupkg-version.outputs.nuget_pkg_version }}
```

Comment:
We can call this `pacakge_version` as this is the version not only for the NuGet package but the SDK in general.